snet_cli/arguments.py

```diff
@@ -249,6 +254,74 @@ def add_contract_options(parser):
         add_contract_function_options(contract_p, contract_name)
 
 
+def _add_service_publish_arguments(parser):
+    parser.set_defaults(fn="publish")
+    parser.add_argument("--no-register", action="store_true", help="does not register the published service")
+    parser.add_argument("--config", metavar="CONFIG_FILE", help="custom service.json file")
+    add_transaction_arguments(parser)
+
+
+def _add_service_update_arguments(parser):
+    parser.set_defaults(fn="update")
+    parser.add_argument("--new-price", help="new price to call the service", type=type_converter("uint256"))
+    parser.add_argument("--new-endpoint", metavar="ENDPOINT", help="new URL to call to reach the service")
+    parser.add_argument("--new-description", metavar="DESCRIPTION", help="new description for the service")
+    parser.add_argument("--config", metavar="CONFIG_FILE", help="custom service.json file")
+    add_transaction_arguments(parser)
+
+
+def add_service_options(parser, config):
+    parser.set_defaults(cmd=ServiceCommand)
+
+    subparsers = parser.add_subparsers(title="services", metavar="COMMAND")
```

Comment:
super small but `title="service commands` would be more consistent; basically the help then looks something like

```
snet service [-h] COMMAND ...

service commands:
  COMMAND
    init
    publish
    update

```

This is currently closer to the "contract" options, in which the list is actually a list of contracts (whereas this is a list of service commands)

Reply:
Fixed